apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -4,7 +4,7 @@
   var MoreAppsNavigation = {
     items: null,
     selectedElemIndex: 0,
-
+    personalizeMode: false,
```

Comment:
What about to declare this variable in app.js? As I understand this variable are used in moreAppsNavigation class and in pinAppsNavigation class.